api/api-stage-operations-v1/src/main/java/com/thoughtworks/go/apiv1/stageoperations/StageOperationsControllerV1Delegate.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.apiv1.stageoperations;
+
+import com.thoughtworks.go.api.ApiController;
+import com.thoughtworks.go.api.ApiVersion;
+import com.thoughtworks.go.api.spring.ApiAuthenticationHelper;
+import com.thoughtworks.go.api.util.HaltApiResponses;
+import com.thoughtworks.go.api.util.MessageJson;
+import com.thoughtworks.go.server.service.ScheduleService;
+import com.thoughtworks.go.spark.Routes;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.http.HttpStatus;
+import spark.Request;
+import spark.Response;
+
+import java.util.Arrays;
+
+import static java.lang.String.format;
+import static spark.Spark.*;
+
+public class StageOperationsControllerV1Delegate extends ApiController {
```

Comment:
I think we don't need a separate Delegate class now that we've upgraded to Spring 4. Originally, the problem was that a Spring bean wouldn't load if we used lambdas in the class, and would crash at server startup with a cryptic error message.
On my local machine, I tried moving the delegate methods into the Spring class itself, and I was able to run the server and hit the API.

Reply:
@akshaydewan alright, I can move them into the controller and run it through the build. If all passes, that’d be great.